src/transport/raw/MessageHeader.h

```diff
@@ -83,7 +86,7 @@ using ExFlags = BitFlags<uint8_t, ExFlagValues>;
 //  |  4 bit  | 4 bit |  4 bit  |  4 bit   |
 //  +---------+-------+---------+----------|
 //  | version | Flags | encType | reserved |
-static constexpr uint16_t kFlagsMask = 0x0F00;
+static constexpr uint16_t kFlagsMask = 0x0F01;
```

Comment:
Please update the comment above to include what the final bit (assuming kSecure) is.
Without it, this seems to pick up the Flags + some undefined reserved bit.